### PR TITLE
rugby-rp.com

### LINKED ISSUE #124680: rugby-rp.com
### Issue URL (Ads)
 [https://rugby-rp.com/worldrank](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Frugby-rp.com%2Fworldrank)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #124680 

```
! https://rugby-rp.com/worldrank
! Mobile , PC 
! Ad
rugby-rp.com##div[class*="-adarea-"]
! https://rugby-rp.com/2022/07/16/japan/86770
! social
rugby-rp.com##.sns-wrap
```

<details><summary>screenshot</summary>

</details><br/>

### CODE REVIEW

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -39,6 +39,7 @@ photoshopvip.net,wotopi.jp,mag.osdn.jp,phileweb.com,joqr.co.jp,yutura.net,goethe
 !
 !
 !
+rugby-rp.com##div[class*="-adarea-"]
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:42]: Add `rugby-rp.com##.pickup-wrap` too, this is for ad.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:42]: I will fix

Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:42]: done

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
@SKEIDs next time please do PR following by our template.

**Pull request merged**